### DISCUSSION TITLE
About ghidra command line interface

### DISCUSSION THREAD

**Discussion by Beast833** [NationalSecurityAgency/ghidra#7240] [Q&A]
Please provide the complete commands to use Ghidra's CLI interface for decompiling an SO file into C/C++ code and saving it to a specified location on my drive.

**Comment by gemesa** [comment]
In earlier vesions there was a script for this: https://github.com/NationalSecurityAgency/ghidra/issues/574#issuecomment-489666133

But it seems to have been removed. There is an old forked version here though: https://github.com/h4sh5/ghidra-headless-decompile/blob/master/Decompile.java
